### DISCUSSION TITLE
Make areas collapsable/dropdown

### DISCUSSION THREAD

**Discussion by AleXSR700** [home-assistant/frontend#9591] [Dashboard]
Would it be possible to make areas in Overview collapsable?

In the Overview, make areas expandable/collapsable/dropdown.
Meaning, when you see the different areas, you have to scroll through an entire area to reach the next one. On a mobile device this is a bit tidious.

How about making areas expandable/collapsable, so you can select which area to expand and see.

Grafana e.g. offers this very pleasant improvement.

I think it would be a pretty cool improvement and enhance the user experience especially for mobile devices as they show all areas in one column.

Thank you for considering my request.

Alex :)

**Comment by spacegaier** [comment]
I am not sure I understand what you mean. An area is only a single line in the area overview table (assuming you are talking about the config pages) => what should get collapsed in your view? It's rather unclear from your initial post.

**Comment by AleXSR700** [comment]
As stated, I am talking about the "Overview", so Lovelace. Also marked it in the topic. 

When you use the default, auto generated lovelace interface "Overview", the sensors etc. are grouped by area. However, these groups are not collapsable. It would be cool to be able to collapse the areas.

Here is what Grafana looks like and what it would be nice to have in Lovelave/Overview
![Screenshot_20210721-204612_Home Assistant](https://user-images.githubusercontent.com/57034871/126543135-171809a5-428a-4818-a7f9-fef752d6c8b7.jpg)

I can then select whivh area I would like to expand and look at.
